java/client/src/main/java/glide/api/models/commands/ZmpopOptions.java

```diff
@@ -0,0 +1,22 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.SortedSetBaseCommands;
+
+// TODO add @link to ZMPOP when implemented
+/**
+ * Mandatory option for {@link SortedSetBaseCommands#bzmpop(double, String[], ScoreModifier)} and
+ * for {@link SortedSetBaseCommands#bzmpop(double, String[], ScoreModifier, long)}.
+ */
+public interface ZmpopOptions {
+    /** Redis API keyword used to extract specific count of elements. */
+    String COUNT = "COUNT";
```

Comment:
Is this the only command that uses Count if not I propose you put it it in the SortedSetBaseCommands interface.